src/_variables.scss

```diff
@@ -271,4 +278,3 @@ $carousel-s2-tablist-thumbnail-active-border-width: 10px !default;
 $txthl-bg-color: #ff0 !default;
 $txthl-color: #000 !default;
 $txthl-font-weight: 700 !default;
-
```

Comment:
Yeah, and it does work because when I remove the line, it adds it back and it still doesn't show in the commit